app/views/casa_cases/show.html.erb

```diff
@@ -75,6 +75,13 @@
       </p>
     <% end %>
 
+    <% if @casa_case.case_court_mandates.exists? %>
+      <h6><strong><%= t(".court_mandates") %>:</strong></h6>
+      <% @casa_case.case_court_mandates.each do |court_mandate| %>
+          <p><%= court_mandate.mandate_text %>, <%= court_mandate.implementation_status.split('_').map(&:capitalize).join(' ') %></p>
```

Comment:
We can use https://apidock.com/rails/v5.2.3/String/humanize here to make it even cooler :)

Reply:
Gotcha, the PR got merged before I got a chance to address the review items.

Reply:
Sorry about that, I was overly eager :) If you have time, https://github.com/rubyforgood/casa/issues/1966 is also available to work on :)